fmf/utils.py

```diff
@@ -10,14 +10,17 @@
 import time
 import warnings
 from io import StringIO
-from typing import Any, List, NamedTuple
+from typing import NamedTuple
 
-import jsonschema
 from filelock import FileLock, Timeout
 from ruamel.yaml import YAML, scalarstring
 from ruamel.yaml.comments import CommentedMap
 
 import fmf.base
+# F401 utils is entry point, while _compat is being used
+# isort with autopep8 made it to two lines. Ruff needed.
+from fmf._compat_jsonschema import JsonSchemaValidationResult  # noqa: F401
+from fmf._compat_jsonschema import validate_data  # noqa: F401
```

Comment:
Why didn't `fmf._compat.jsonschema` work?

Reply:
hmm, I don't remember. Any suggestion about how the structure should look like are welcome.

Reply:
The design I saw was have a `_compat` package and under that the same structure as the most recent interface that will eventually replace it, e.g. `_compat.collections.abc` for `Itterator`, `_compat.warnings` for `deprecated` etc. That way the eventual change is to simply drop the `_compat` from the `from import`.

The part that I didn't see used was something like `from _compat import warnings` that would cover both compatibilities like `warnings.deprecated` and normal functions like `warnings.warn`. It seems sensible to me if you want to eventually use the fully-qualified naming, but not sure if there are pitfalls that I'm overlooking.

Reply:
Lgtm, don't think we can follow the other comoat designs here, but at least the change is localized so it helps to be in this form 👍